### ISSUE TITLE
Frametimes does not look great in raw with PresentMon Beta

### ISSUE THREAD

**Issue by TheWonderfulTartiflette** [GameTechDev/PresentMon#166]
![image](https://github.com/GameTechDev/PresentMon/assets/44057002/db345599-0fb4-42d9-b586-4ea55b1fed8c)
Look at this. The frametimes seems to go all over the place, when in reality the game is very stable (God of War). For comparison, here's the frametime graph of Afterburner : 
![image](https://github.com/GameTechDev/PresentMon/assets/44057002/2edeea3b-2d75-445c-b09e-037dfab9c504)

I don't know what's causing PresentMon Beta's graphs to look horrible, but I think this really should be fixed, or at least make it display in an Afterburner's style

**Comment by AlexUnwinder** [comment]
Give it a read, pay attention to Tom's slide from presentation with my comments:

https://forums.guru3d.com/threads/msi-ab-rtss-development-news-thread.412822/page-206#post-6159952

RTSS and PresentMon calculate frametimes on slightly different time points. RTSS samples it as Start-to-Start (and can be changed to Present-to-Present depending on your preferences) as it is closer to game engine's frame timings, PresentMon samples it as Present-to-Presnet only.

**Comment by TheWonderfulTartiflette** [comment]
I've read it, but I won't lie I didn't understand everything. Tldr, is this
normal that this is happening? And how do I make it look like RTSS'?

Le ven. 1 sept. 2023, 19:45, AlexUnwinder ***@***.***> a
écrit :

**Comment by JeffersonMontgomery-Intel** [comment]
As Alex pointed out, the beta is calculating "Frame Time" incorrectly, which is leading to these less-stable values.  It will be fixed for the next release.  Thanks for pointing this out.

**Comment by TheWonderfulTartiflette** [comment]
@JeffersonMontgomery-Intel Thanks a ton. By the way, does the PresentMon Beta software updates itself, or do we have to subscribe to the releases?

**Comment by JeffersonMontgomery-Intel** [comment]
No, for now you'll need to subscribe to hear about new releases and re-install manually.

I'm going to re-open this just to remind me to actually fix it 😁 (or if anybody wonders the same thing until we fix it).

**Comment by TheWonderfulTartiflette** [comment]
@JeffersonMontgomery-Intel seems like I made a mistake, I can't really "subscribe" to releases as PresentMon isn't on GitHub. Where can I know when an update is out then?

**Comment by AlexUnwinder** [comment]
I do not think that beta calculates in incorrectly and needs to be fixed. You just calculate slightly different metric, which is also valid and useful. Presentmon's frametime is msBetweenPresents and RTSS reflects something like msBetweenPresentReturns by default.

**Comment by JeffersonMontgomery-Intel** [comment]
If you go to the main repository page: https://github.com/GameTechDev/PresentMon

Click the "watch" dropdown, then "custom", then you can check "releases" and you should get notified for future releases.

**Comment by TheWonderfulTartiflette** [comment]
Yeah but that's only for the PresentMon SDK, right? So no PresentMon Beta GUI?

**Comment by JeffersonMontgomery-Intel** [comment]
No, we'll post/announce new GUI beta releases here too.

**Comment by TheWonderfulTartiflette** [comment]
Okay, thanks a ton! :)
